server/server-communication.web.js

```diff
@@ -52,6 +52,7 @@ class SocketServer {
 
   listen() {
     server.listen(this.port);
+    console.log(`server listening on port ${this.port}`);
```

Comment:
you mind changing this to include the full URL so i can click it from the terminal 😛